### DISCUSSION TITLE
import fails when source and destination is the same

### DISCUSSION THREAD

**Discussion by timaschew** [firefly-iii/firefly-iii#6397] [Firefly III]
I'm trying to import a CSV file with 1300 entries.
After conversion I start the import job via the webinterface of FIDI and then it shows me this:

```
 The import routine has finished 🎉. Done!
Error(s)

    Line #49:
        (2)transactions.0.destination_id: Source and destination are the same. (original value: "1")
        (2)transactions.0.destination_name: Source and destination are the same. (original value: "(not found)")
```

Here is the source in the CSV file:
```
17.12.2019;17.12.2019;ING DIBA;Gutschrift;Gutschrift vom 17.12.2019;11.232,42;EUR;10,50;EUR
```

The name of my default checking account is `ING DiBa`, I guess it's case in-sensitive and I need to use a different name in the  CSV file. Of course it's not the same account. It's a transaction between an account which is owner by the bank itself and an account wich I own. It would be nice if that could be detected during conversion and before the import job is started or is that not possible?

**Comment by JC5** [comment]
So are there two instances of `ING DiBa`? One is yours, and the other is the bank itself?

**Reply by timaschew** [reply]
It should be two instances, but it's mapped to one, that's the problem.

I've created a checking acount with the name `ING DiBa`.

The CSV file has an entry where the bank sent me 10,50 € and coincidence the name is also `ING DIBA` which seems to be mapped automatically to my checking account which is wrong!

I've renamed the account in the CSV file to `ING DIBA ZENTRALE` and now it works.

**Reply by JC5** [reply]
Ah yes, that's problematic indeed. I have a similar thing with my own bank. In theory it's avoidable if there is an expense account called "ING diba", but i have to check it out later.
